lib/dor/models/concerns/embargoable.rb

```diff
@@ -17,10 +17,12 @@ module Embargoable
     def release_embargo(release_agent = 'unknown')
       embargo_service.release(release_agent)
     end
+    deprecation_deprecate release_embargo: 'Use the method in EmbargoService instead'
```

Comment:
is this advising use of the deprecated method in `EmbargoService`?  if so, why not just give the same warning as that method, to reduce warning indirection?  if that's not what's happening, apologies for misunderstanding.

Reply:
Yeah, this was 2 commits, and this message made sense with the first, but not after the second.